ddtrace/settings/config.py

```diff
@@ -803,12 +803,17 @@ def _format_tags(self, tags: List[Union[str, Dict]]) -> Dict[str, str]:
     def enable_remote_configuration(self):
         # type: () -> None
         """Enable fetching configuration from Datadog."""
+        from ddtrace.internal.flare.flare import Flare
+        from ddtrace.internal.flare.handler import _handle_tracer_flare
+        from ddtrace.internal.flare.handler import _tracerFlarePubSub
         from ddtrace.internal.remoteconfig.worker import remoteconfig_poller
 
         remoteconfig_pubsub = self._remoteconfigPubSub()(self._handle_remoteconfig)
+        flare = Flare(trace_agent_url=self._trace_agent_url, api_key=self._dd_api_key)
+        tracerflare_pubsub = _tracerFlarePubSub()(_handle_tracer_flare, flare)
         remoteconfig_poller.register("APM_TRACING", remoteconfig_pubsub)
-        remoteconfig_poller.register("AGENT_CONFIG", remoteconfig_pubsub)
```

Comment:
I'm surprised that this change doesn't break anything. I'd expect changing the handler for `AGENT_CONFIG` to have a meaningful effect, because it's presumably already used, separately from the flare. Just a flag - please tell me if I'm missing information that makes this change ok.

Reply:
I think this switch isn't breaking because these changes won't go into effect until the app is restarted/ddtrace is updated? I haven't actually tried to test an ongoing app upgrading to a version that has this change, but I didn't think it would be a risk because `enable_remote_configuration` only runs when ddtrace is starting up. If that's an incorrect assumption, then it might be something I need to look into though

Reply:
I don't think I understand this change. are we no longer going to call `remoteconfig_pubsub` on `AGENT_CONFIG` changes? is that a problem?

I am having trouble tracing the code to understand what we do from `AGENT_CONFIG` events.

Reply:
Could settings changes come in through `APM_TRACING` or `AGENT_CONFIG`?